source/_docs/installation/virtualenv.markdown

```diff
@@ -52,7 +52,7 @@ _(If you're on a Debian based system, you will need to install Python virtual en
     ```
     $ cd homeassistant
     ```
- 3. Activate the virtual environment:
+ 3. Activate the virtual environment (typically /srv/homeassistant/):
```

Comment:
This guide doesn't tell to setup the `venv` at `/srv/homeassistant`. I think that now it's misleading because in Step 3 the user should already be in the right directory. The Rasbian guide and Hassbian uses that directory.